src/plugins/lua/CMakeLists.txt

```diff
@@ -36,7 +36,7 @@ add_plugin (lua
 if (ADDTESTING_PHASE)
 	# bindings are required for tests
 	list (FIND BINDINGS "swig_lua" FINDEX)
-	if (NOT BUILD_SHARED AND BUILD_TESTING AND FINDEX GREATER -1)
```

Comment:
Unfortunately the [tests of the Lua plugin](https://travis-ci.org/ElektraInitiative/libelektra/jobs/314400163#L5796) do not work on the Mac if we install Elektra. Maybe we should disable them if we notice that 

- the current machine is a Mac and
- Elektra is already installed

.

Reply:
Ahh, it seems they only fail on already installed Elektra. This likely means that the plugin is actually dysfunctional, so its not only a matter of disabling the test but the complete plugin for MacOSX?

Reply:
I guess you are right. If I enable `BUILD_STATIC` and `BUILD_FULL` and disable `BUILD_SHARED`, then building the Lua (and Python) plugins does not work at all on my machine.